tests/integration.rs

```diff
@@ -1109,3 +1110,21 @@ fn test_client() -> Result<()> {
         .with_context(|| "failed execute client.name")?;
     Ok(())
 }
+
+#[test]
+fn test_filter() -> Result<()> {
+    let port = 6508;
+    let _guards =
+        vec![start_valkey_server_with_module("filter", port)
+            .with_context(|| FAILED_TO_START_SERVER)?];
+    let mut con = get_valkey_connection(port).with_context(|| FAILED_TO_CONNECT_TO_SERVER)?;
+    // currently the example implements a filter for info command
+    redis::cmd("info")
```

Comment:
From the example module, we replace the command with info2. So, should the response returned be "info2" ?

Reply:
good point, added test